google/generativeai/version.py

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 from __future__ import annotations
 
-__version__ = "0.6.0"
+__version__ = "0.7.0"
```

Comment:
This is fine but for future reference we try to keep the version increments in their own change. Including them in other changes can make cherry picking / rebasing real difficult.